src/shared-with-pluot-core/Util.js

```diff
@@ -0,0 +1,27 @@
+export const getParticipantIsSubscribedToTrack = (state, id, mediaTag) => {
```

Comment:
it this `Util.js` file meant to house shared redux selectors? if so, the name could reflect that.

Reply:
I had thought of it as a catch-all. Do you think it's better to separate out redux selectors into their own file (which would be this file, for now), @kompfner ?

Reply:
renamed